### PR TITLE
create Player Table

### PR DESCRIPTION
**Pull request by saifjilani**
### Problem

Need a Player Model to keep track of players in the game
### Solution

Generate a Player Model

### CODE REVIEW

File: db/migrate/20160811030257_create_players.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreatePlayers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :players do |t|
+      t.string :first_name
+      t.string :last_name
+
+      t.timestamps null: false
```
Comment (muneebaahmad) [inline db/migrate/20160811030257_create_players.rb:7]: why do we have timestamps for when the player was created? and what does the false mean?

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160811030257) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "players", force: :cascade do |t|
```
Comment (muneebaahmad) [inline db/schema.rb:23]: doesn't it tell us not to edit this file?

**Comment by muneeb1993** [comment]
(y)

**Comment by muneebaahmad** [comment]
good stuff

File: db/migrate/20160811030257_create_players.rb (hunk 1 above)
Comment (saifjilani) [inline db/migrate/20160811030257_create_players.rb:7]: @muneeb1993 timestamps are created by default for all DB tables

**Pull request merged**